tasks/eth/ink-001-permissionless-proofs/VALIDATION.md

```diff
@@ -0,0 +1,46 @@
+# Validation
```

Comment:
the `.env` is using the 
```
SAFE_NONCE=6
```
We should expect some storage validation about the safe state transition.